docs/syntax_and_semantics/c_bindings/struct.md

```diff
@@ -41,6 +41,24 @@ lib C
 end
 ```
 
+Structs that are defined inside a `lib` can be included as modules internally in other `lib` defined structs, for example:
```

Comment:
**polish:** This is only a tiny detail, might be better to not framt it "include _as_ a module" but "include _like_ a module". The struct is still a struct, not a module.

```suggestion
Structs that are defined inside a `lib` can be included, like modules, internally in other `lib` defined structs, for example:
```